src/main/java/org/pih/petl/job/schedule/PetlScheduledExecutionTask.java

```diff
@@ -44,14 +48,15 @@ public void execute(JobExecutionContext jobExecutionContext) throws JobExecution
         if (enabled && !inProgress) {
             try {
                 inProgress = true;
+                Schedule globalSchedule = applicationConfig.getSchedule();  // get the global schedule, if configured
                 Date currentDate = jobExecutionContext.getFireTime();
                 log.debug("Executing Task: " + currentDate);
                 Map<String, PetlJobConfig> jobs = etlService.getAllConfiguredJobs();
                 log.debug("Found " + jobs.size() + " configured Jobs");
                 for (String jobPath : jobs.keySet()) {
                     log.debug("Checking job: " + jobPath);
                     PetlJobConfig jobConfig = jobs.get(jobPath);
-                    Schedule schedule = jobConfig.getSchedule();
+                    Schedule schedule = jobConfig.getSchedule() != null ? jobConfig.getSchedule() : globalSchedule;  // override global schedule with any local job-specific schedule
```

Comment:
Not sure the inline comment here is necessary. Matter of taste, though.

Reply:
Fair enough